### ISSUE TITLE
Plugin emits error if all tslint issues are fixed.

### ISSUE THREAD

**Issue by mxl** [panuhorsmalahti/gulp-tslint#120]
Plugin emits error even If all reported issues are auto fixed by tslint.

gulp-tslint version: 8.0.0
tslint version: 5.2.0
Operating system: Debian

**Comment by panuhorsmalahti** [comment]
Can you provide more information? E.g. your configuration? Even better would be a repository that can be pulled

**Comment by mxl** [comment]
Sorry, I do not have enough time to prepare example project right now. And the issue is not configuration issue. 

tslint returns [LintResult.fixes](https://github.com/palantir/tslint/blob/a3d8d6fd5a52537e7d61f2759b68980c649a7685/src/index.ts#L41) array. gulp-tslint should [somewhere](https://github.com/panuhorsmalahti/gulp-tslint/blob/19726ad3c51663f41e6089441e435b624159a67a/index.ts#L199) check if all errors were fixed (something like `failures.length < fixes.length`) and should not emit error in that case.

**Comment by mxl** [comment]
In my project I use plugin like:
```javascript
var gulp = require('gulp),
      gulpTsLint = require('gulp-tslint'),
      tslint = require('tslint');
      
gulp.task('tslint', function () {
    var program = tslint.Linter.createProgram('tsconfig.json');
    return gulp.src('src/**/*.ts')
        .pipe(gulpTsLint({
            configuration: 'tslint.json',
            fix: true,
            formatter: 'prose',
            program: program,
            tslint: tslint
        }))
        .pipe(gulpTsLint.report());
});
```

tslint.json is:

```json
{
  "rules": {
    "class-name": true,
    "curly": true,
    "eofline": true,
    "forin": true,
    "indent": [
      true,
      "spaces"
    ],
    "label-position": true,
    "max-line-length": [
      true,
      120
    ],
    "no-arg": true,
    "no-bitwise": true,
    "no-console": [
      true,
      "debug",
      "info",
      "time",
      "timeEnd",
      "trace"
    ],
    "no-construct": true,
    "no-debugger": true,
    "no-duplicate-variable": true,
    "no-empty": false,
    "no-eval": true,
    "no-string-literal": false,
    "no-trailing-whitespace": true,
    "no-use-before-declare": true,
    "one-line": [
      true,
      "check-open-brace",
      "check-catch",
      "check-else",
      "check-whitespace"
    ],
    "quotemark": [
      true,
      "single"
    ],
    "radix": true,
    "semicolon": [
      true,
      "never"
    ],
    "triple-equals": [
      true,
      "allow-null-check"
    ],
    "variable-name": false,
    "whitespace": [
      true,
      "check-branch",
      "check-decl",
      "check-operator",
      "check-separator"
    ]
  }
}
```

As an example you can use following code:

```typescript
const a = 1;
const b = 2;
```

It will emit error because of semicolon rule failures though they are autofixed.

**Comment by panuhorsmalahti** [comment]
Thank you, that's detailed enough! I missed the word "auto" in your post so I thought it could've been a configuration issue.
